spec.bs

```diff
@@ -1326,6 +1327,24 @@ Issue: This would ideally be replaced by a more descriptive algorithm in Infra.
 
 </div>
 
+<div algorithm>
+To <dfn>round a value</dfn> given a {{double}} |value|:
+  1. If |value| is not a [=valid floating-point number=], return |value|.
+  1. Let |valueExp| be |value|'s IEEE 754 biased exponent field minus 1023.
+  1. Let |normValue| be |value| multiplied by 2 raised to the power of -1 multiplied by |valueExp|.

```

Comment:
For this mathematical statement, and actually all of the ones more generally in this algorithm, I think it would be better to just be explicit with the math symbols. There is precedent for this in for example, https://url.spec.whatwg.org/#:~:text=If%20the%20last%20item%20in%20numbers,Increment%20counter%20by%201 where explicit symbols are used to write the actual math out. Can you do that here?